app/assets/javascripts/autocomplete.coffee

```diff
@@ -0,0 +1,54 @@
+delay = do ->
+  timer = 0
+  (callback, ms) ->
+    clearTimeout timer
+    timer = setTimeout(callback, ms)
+    return
+
+update_textfield = (list_element) ->
+  $(list_element).removeClass('suggestion-focused')
+  $('.js-autocomplete-textfield').val($(list_element).find('.res-name')[0].innerHTML)
```

Comment:
That sounds fine to me, we have validations that the repo belongs to the user on the controller so I'm not worried about anyone spoofing us.